worlds/yosemite.world

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" ?>
+<sdf version="1.5">
+  <world name="default">
+    <scene>
+      <sky>
+        <clouds>
+          <speed>12</speed>
+        </clouds>
+      </sky>
+    </scene>
+    <!-- A global light source -->
+    <include>
+      <uri>model://sun</uri>
+    </include>
+    <include>
+      <uri>model://asphalt_plane</uri>
+    </include>
```

Comment:
Is the asphalt_plane required? It looks a little strange with the rest of the terrain: [video](https://youtu.be/7bAz36IsBko)

Reply:
I don't think it makes sense.

Reply:
It was there to support our demo allowing parametric launching of N vehicles and when spawning them it was much easier to have a flat plane to target. Removing it's no problem, you just have to be more careful about where you spawn is flat enough.